### ISSUE TITLE
Unable to update deal's custom field because of wrong fieldname normalization

### ISSUE THREAD

**Issue by ubaranouski-pl** [pipedrive/client-nodejs#297]
Custom fields of deal updated by a key. Keys are md5 hashes and normalizer inserts underscore character between letters and numbers. As a result unable to update custom fields.

d1f443f570fc497f809f07309cd3a6903ffe8963
becomes
d_1_f_443_f_570_fc_497_f_809_f_07309_cd_3_a_6903_ffe_8963

`{"success":false,"message":"Bad Request","errorCode":400,"context":{"req":{"method":"PUT","url":"https://api.pipedrive.com/v1/deals/25?api_token=...","data":{"d_1_f_443_f_570_fc_497_f_809_f_07309_cd_3_a_6903_ffe_8963":"Field value text goes here"},"headers":{"user-agent":"Pipedrive-SDK-Javascript-13.1.1","content-type":"application/json","accept":"application/json"}},"header":{"date":"Thu, 27 Jan 2022 17:45:11 GMT","content-type":"application/json","transfer-encoding":"chunked","connection":"close ... ... ... "status":400,"text":"{\"success\":false,\"error\":\"Bad request\",\"error_info\":\"Please check developers.pipedrive.com for more information about Pipedrive API.\",\"data\":null,\"additional_data\":null}"},"errorInfo":"Please check developers.pipedrive.com for more information about Pipedrive API."}`

**Comment by RuTsEST** [comment]
Oh no!
I've create a bug report for it and we'll look at it ASAP.

**Comment by alex-all3dp** [comment]
Also seeing this issue when creating deals with the custom fields pre-filled

**Comment by Reivilo85k** [comment]
@ubaranouski-pl Would it be ok for you to share more details about the request you made to generate this error please ? I have been looking into it and cannot reproduce it so far.

**Comment by chinovader** [comment]
Hello, same issue here.

This is an example to add an organization:

```
(async () => {
  const data = {
      "name": "Test Name",
      "address": "Test Address",
      "42dabde1107c98c6f91ece71ce44c9ff51d6eb08": "Test value",
  };

  try {
    console.log('Adding organization...');
    const api = new pipedrive.OrganizationsApi();
    const response = await api.addOrganization(data);
    console.log('Organization was added successfully', response);
  } catch (err) {
    const errorToLog = err.context?.body || err;
    console.log('Adding failed', errorToLog);
  }
})();
```
The response is:

```
Adding organization...
Adding failed {
  success: false,
  error: 'Invalid field(s) in the payload: 42_dabde_1107_c_98_c_6_f_91_ece_71_ce_44_c_9_ff_51_d_6_eb_08',
  error_info: 'Consulta developers.pipedrive.com para obtener más información sobre el API de Pipedrive.',
  data: null,
  additional_data: null
}

```

**Comment by Reivilo85k** [comment]
Thanks for reporting the bug and for additional info, we have found the culprit and fixed it. It was indeed related to what @chinovader has described 🥇 

The issue has been fixed in the latest patch 13.2.2

**Comment by matyushen** [comment]
This is still (v18.1.3) an issue? Getting this error for `4042d46f305337b4d0d72f0cdc6dd16770ff02d2_postal_code` (part of the Address field type) field:

````
\"{\\\"success\\\":false,\\\"error\\\":\\\"Invalid field(s) in the payload: 4042_d_46_f_305337_b_4_d_0_d_72_f_0_cdc_6_dd_16770_ff_02_d_2_postal_code\\\",\\\"error_info\\\":\\\"Please check developers.pipedrive.com for more information about Pipedrive API.\\\",\\\"data\\\":null,\\\"additional_data\\\":null}\"
````
